crates/kubelet/src/kubelet.rs

```diff
@@ -63,8 +69,53 @@ impl<T: 'static + Provider + Sync + Send> Kubelet<T> {
             }
         });
 
-        // This informer listens for pod events.
-        let provider = self.provider.clone();
+        // TODO: How should we configure this value? We should eventually have a max pods setting
+        // just like a normal kubelet, so maybe that?
+        let (error_sender, mut error_receiver) = mpsc::channel::<(KubePod, anyhow::Error)>(200);
+        let client_clone = client.clone();
+        let error_handler = tokio::task::spawn(async move {
+            let client = client_clone;
+            while let Some(e) = error_receiver.recv().await {
+                let (pod, err) = e;
+                let json_status = serde_json::json!(
+                    {
+                        "metadata": {
+                            "resourceVersion": "",
+                        },
+                        "status": {
+                            "phase": Phase::Failed,
+                            "message": format!("{}", err),
```

Comment:
Is the `format!` necessary?

Reply:
Oh, I guess it will just get serialized, so no.

Reply:
Looks like anyhow doesn't implement serialize in a way that serde likes